.github/workflows/release.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
     - name: Install pip dependencies
       if: steps.cache.outputs.cache-hit != 'true'
       run: |
-        pip install .[datasets,tests]
+        pip install .[datasets,tests] tensorboard
```

Comment:
This only needs to be added to the notebooks test below, you can remove the other two additions.